release/config/sing-box.service

```diff
@@ -7,7 +7,7 @@ After=network.target nss-lookup.target
 WorkingDirectory=/var/lib/sing-box
 CapabilityBoundingSet=CAP_NET_ADMIN CAP_NET_BIND_SERVICE
 AmbientCapabilities=CAP_NET_ADMIN CAP_NET_BIND_SERVICE
-ExecStart=/usr/bin/sing-box run -c /etc/sing-box/config.json
```

Comment:
So we may need edit `/release/local/install.sh` ? 
`sudo cp $(go env GOPATH)/bin/sing-box /usr/local/bin/`
I just want the two files to be compatible so that the install script is useable.